SharedSources/MediaLibraryModel/MediaLibraryBaseModel.swift

```diff
@@ -49,6 +50,10 @@ extension MLBaseModel {
         return MovieCollectionViewCell.self
     }
 
+    var editCellType: BaseCollectionViewCell.Type {
```

Comment:
I'm not sure here but do we really need this property inside `MediaLibraryBaseModel`? I'm wondering if this is the right place/right level of abstraction.

Reply:
I wouldn't know where else to be honest since the UI will be different for albums and playlists I guess